core/tests/test_apps.bitcoin.address.py

```diff
@@ -1,18 +1,20 @@
-from common import *
-from trezor.crypto import bip32, bip39
-from trezor import wire
-from trezor.messages import GetAddress
-from trezor.enums import InputScriptType
-from trezor.utils import HashWriter
-
-from apps.common import coins
 from apps.bitcoin.addresses import *
 from apps.bitcoin.addresses import (
-    _address_p2wsh, _address_p2wsh_in_p2sh,
-    _address_multisig_p2wsh_in_p2sh, _address_multisig_p2sh
+    _address_multisig_p2sh,
+    _address_multisig_p2wsh_in_p2sh,
+    _address_p2wsh,
+    _address_p2wsh_in_p2sh,
 )
 from apps.bitcoin.keychain import validate_path_against_script_type
 from apps.bitcoin.writers import *
+from apps.common import coins
+from trezor import wire
+from trezor.crypto import bip32, bip39
+from trezor.enums import InputScriptType
+from trezor.messages import GetAddress
+from trezor.utils import HashWriter
+
+from common import *
```

Comment:
we actually need "common" to be first. but maybe this is not worth worrying about, just reverting the whole of `core/tests` is good enough